koku/masu/processor/ocp/ocp_cloud_parquet_summary_updater.py

```diff
@@ -247,8 +261,8 @@ def update_aws_summary_tables(self, openshift_provider_uuid, aws_provider_uuid,
                 end_date,
             )
 
-            aws_bill_ids = [str(bill.id) for bill in aws_bills]
-            current_aws_bill_id = aws_bills.first().id if aws_bills else None
+            aws_bill_ids = [bill.id for bill in aws_bills]
+            current_aws_bill_id = aws_bill_ids[0] if aws_bills else None
```

Comment:
```suggestion
            current_aws_bill_id = aws_bill_ids[0]
```
The if else None can be removed now because of the check above.